app/parsers/markdown-parser.rb

```diff
@@ -0,0 +1,166 @@
+require 'redcarpet'
+
+# TODO: buradan asagidaki fonksiyonlar standartlar belirlenerek duzenlenecek
+# TODO: ic ice döngüler sırasında parse işlemi sorun vermekte. Düzeltilmesi gerekiyor
+
+# öyak çeviri yönetim sistemi için kullanılabilecek standartlarda render
+class OyakRender < Redcarpet::Render::HTML
```

Comment:
Sınıf adı Öyak'a özgü bir isim değil de ileride belki de ayrı bir gem olma potansiyeli bulunduğu için daha genel ve açıklayıcı bir isim tercih edilebilir.